src/libraries/Microsoft.PowerFx.Interpreter/Functions/CommonErrors.cs

```diff
@@ -59,6 +59,16 @@ public static ErrorValue InvalidNumberFormatError(IRContext irContext)
             });
         }
 
+        public static ErrorValue InvalidBooleanFormatError(IRContext irContext)
+        {
+            return new ErrorValue(irContext, new ExpressionError()
+            {
+                Message = "The Boolean could not be parsed",
```

Comment:
Maybe: The value passed to the Boolean function could not be interpreted as a boolean?

Not sure if "parsed" is a verb we expect our customers to be familiar with

Reply:
Changed